prime-router/src/main/kotlin/cli/tests/BasicTests.kt

```diff
@@ -66,172 +66,94 @@ class Ping : CoolTest() {
 }
 
 /**
- * End to end test for the universal pipeline. This test should read in a submission, verify it reached all steps
- * (receive, convert, route, translate, batch, send) and that it generated the correct lineage records.
+ * Updated end2end test for the universal pipeline. This test posts a report and verifies that the correct
+ * receiver received the file and that the correct file is in the sftp directory.
+ * This test is repeated: Once for the normal UP workflow and once for the ELIMS workflow.
  */
 class End2EndUniversalPipeline : CoolTest() {
```

Comment:
I considered renaming the test name and commands, but I couldn't think of any conflict with keeping them the same.

Reply:
Discussed in standup: We are going to keep the old test as removed.